src/script/server/api/interfaces/connect.js

```diff
@@ -0,0 +1,30 @@
+const log = require('loglevel')
+
+const errorHandler = callback => async (req, res, ...args) => {
+  try {
+    const content = await callback(req, res, ...args)
+    if (content) {
+      res.send(content)
+    } else {
+      res.sendStatus(200)
+    }
+  } catch (ex) {
+    log.error(ex)
+    res.status(500).send({ ex })
```

Comment:
Error prototype is serialized as empty object.
To have error info in response, we need to do something like that :
```
const { name, message } = ex
res.status(500).send({ name, message })
```